providers/fetcher/githubProcessor.js

```diff
@@ -981,8 +1000,8 @@ class GitHubProcessor {
     request.linkResource(relation.origin, `${qualifier}`);
     request.linkSiblings(`${relation.qualifier}:pages`);
     request.linkCollection('unique', `${relation.qualifier}:pages:${relation.guid}`);
-    const urnPrefix = relation.type === 'commit' && qualifier.includes('pull_request') ? `repo:${qualifier.split(':')[2]}:pull_request_commit` : relation.type;
-    const id = relation.type === 'commit' ? 'sha' : 'id';
+    const urnPrefix = relation.type === 'pull_request_commit' ? `repo:${qualifier.split(':')[2]}:pull_request_commit` : relation.type;
```

Comment:
Interesting, pullreqeust commits are no longer listed in a relation (see change on line 300 (in new revision).  So do we even need this test any more?